Description: Set the source encoding to fix the build failure with recent JDKs
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/src/org/javacc/parser/build.xml
+++ b/src/org/javacc/parser/build.xml
@@ -47,6 +47,7 @@
            debug="${build.compiler.debug}"
            source="1.4"
            target="1.4"
+           encoding="ISO-8859-1"
            classpath="${javacc.root}/classes:${junit.jar}">
     </javac>
   </target>
--- a/build.xml
+++ b/build.xml
@@ -183,7 +183,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/CORBA-IDL examples/CORBA-IDL/IDL.jj" />
     </java>
-    <javac srcdir="test.tmp/CORBA-IDL" />
+    <javac srcdir="test.tmp/CORBA-IDL" encoding="ISO-8859-1"/>
 
     <echo />
     <copy todir="test.tmp/GUIParsing/ParserVersion">
@@ -194,7 +194,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/GUIParsing/ParserVersion examples/GUIParsing/ParserVersion/CalcInput.jj" />
     </java>
-    <javac srcdir="test.tmp/GUIParsing/ParserVersion" />
+    <javac srcdir="test.tmp/GUIParsing/ParserVersion" encoding="ISO-8859-1"/>
 
     <echo />
     <copy todir="test.tmp/GUIParsing/TokenMgrVersion">
@@ -205,7 +205,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/GUIParsing/TokenMgrVersion examples/GUIParsing/TokenMgrVersion/CalcInput.jj" />
     </java>
-    <javac srcdir="test.tmp/GUIParsing/TokenMgrVersion" />
+    <javac srcdir="test.tmp/GUIParsing/TokenMgrVersion" encoding="ISO-8859-1"/>
 
     <echo />
     <copy todir="test.tmp/Interpreter">
@@ -219,7 +219,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/Interpreter test.tmp/Interpreter/SPL.jj" />
     </java>
-    <javac srcdir="test.tmp/Interpreter" />
+    <javac srcdir="test.tmp/Interpreter" encoding="ISO-8859-1"/>
   	
     <echo />
     <copy todir="test.tmp/JJTreeExamples.eg1">
@@ -235,7 +235,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/JJTreeExamples.eg1 test.tmp/JJTreeExamples.eg1/eg1.jj" />
     </java>
-    <javac srcdir="test.tmp/JJTreeExamples.eg1" />
+    <javac srcdir="test.tmp/JJTreeExamples.eg1" encoding="ISO-8859-1"/>
   	
     <echo />
     <copy todir="test.tmp/JJTreeExamples.eg2">
@@ -251,7 +251,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/JJTreeExamples.eg2 test.tmp/JJTreeExamples.eg2/eg2.jj" />
     </java>
-    <javac srcdir="test.tmp/JJTreeExamples.eg2" />
+    <javac srcdir="test.tmp/JJTreeExamples.eg2" encoding="ISO-8859-1"/>
   	
     <echo />
     <copy todir="test.tmp/JJTreeExamples.eg3">
@@ -267,7 +267,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/JJTreeExamples.eg3 test.tmp/JJTreeExamples.eg3/eg3.jj" />
     </java>
-    <javac srcdir="test.tmp/JJTreeExamples.eg3" />
+    <javac srcdir="test.tmp/JJTreeExamples.eg3" encoding="ISO-8859-1"/>
   	
     <echo />
     <copy todir="test.tmp/JJTreeExamples.eg4">
@@ -281,13 +281,13 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/JJTreeExamples.eg4 test.tmp/JJTreeExamples.eg4/eg4.jj" />
     </java>
-    <javac srcdir="test.tmp/JJTreeExamples.eg4" />
+    <javac srcdir="test.tmp/JJTreeExamples.eg4" encoding="ISO-8859-1"/>
 
     <echo />
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/JavaCCGrammar examples/JavaCCGrammar/JavaCC.jj" />
     </java>
-    <javac srcdir="test.tmp/JavaCCGrammar" />
+    <javac srcdir="test.tmp/JavaCCGrammar" encoding="ISO-8859-1"/>
 
 	<!-- TODO, test cases for JavaGrammars -->
 
@@ -300,7 +300,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/MailProcessing examples/MailProcessing/Faq.jj" />
     </java>
-    <javac srcdir="test.tmp/MailProcessing" />
+    <javac srcdir="test.tmp/MailProcessing" encoding="ISO-8859-1"/>
 
     <echo />
     <copy todir="test.tmp/Obfuscator">
@@ -317,7 +317,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/Obfuscator examples/Obfuscator/MapFile.jj" />
     </java>
-    <javac srcdir="test.tmp/Obfuscator" />
+    <javac srcdir="test.tmp/Obfuscator" encoding="ISO-8859-1"/>
 
 	<!-- TODO, test cases for SimpleExamples -->
   	
@@ -333,7 +333,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/Transformer test.tmp/Transformer/ToyJava.jj" />
     </java>
-    <javac srcdir="test.tmp/Transformer" />
+    <javac srcdir="test.tmp/Transformer" encoding="ISO-8859-1"/>
 
     <echo />
     <copy todir="test.tmp/VTransformer">
@@ -347,7 +347,7 @@
     <java fork="true" classname="javacc" classpath="bin/lib/javacc.jar">
       <arg line="-OUTPUT_DIRECTORY=test.tmp/VTransformer test.tmp/VTransformer/Java1.1.jj" />
     </java>
-    <javac srcdir="test.tmp/VTransformer" />
+    <javac srcdir="test.tmp/VTransformer" encoding="ISO-8859-1"/>
 
   </target>
 </project>
